config/locales/en.yml

```diff
@@ -1,33 +0,0 @@
-# Files in the config/locales directory are used for internationalization
```

Comment:
Let's put all translations in this file, you can create one for the `es` translation as well